featuremanagement/_featuremanager.py

```diff
@@ -104,6 +104,8 @@ def is_enabled(self, feature_flag_id, **kwargs):
                 else:
                     if self._filters[filter_name].evaluate(feature_filter, **kwargs):
                         return True
+            else:
+                raise ValueError("Feature flag {} has unknown filter {}".format(feature_flag_id, filter_name))
```

Comment:
Not required here, but in .NET we offer an option that doesn't throw when this happens. ( IgnoreMissingFeatureFilters ), we just log in when it happens. By default it throws though.

Reply:
Maybe a feature to add later, in Spring this falls under the "fail-fast" option, but such a thing doesn't exist or is atypical in Python.